### PR TITLE
yauheni/76930/sendemailtemplate ts migration

### PR DESCRIPTION
**Pull request by yauheni-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/account/src/Components/sent-email-modal/sent-email-modal.jsx [hunk 1]
```diff
@@ -104,7 +104,6 @@ const SentEmailModal = ({
             txt_resend={localize('Resend email')}
             txt_resend_in={localize('Resend email in')}
             onClickSendEmail={onClickSendEmail}
-            closeEmailModal={onClose}
```
Comment (niloofar-deriv) [inline packages/account/src/Components/sent-email-modal/sent-email-modal.jsx:107]: We don't need this?

Comment (shayan-deriv) [reply packages/account/src/Components/sent-email-modal/sent-email-modal.jsx:107]: @niloo-fs we don't have it in `send-email-template.tsx`

**Review by shayan-deriv — APPROVED** [review]

File: packages/components/src/components/send-email-template/send-email-template.tsx [hunk 2]
```diff
@@ -1,34 +1,45 @@
...
+    const { is_appstore } = React.useContext<{ is_appstore?: boolean }>(PlatformContext);
 
     const timeout_limit = resend_timeout || 60;
-    let resend_interval = null;
+    let resend_interval: number;
 
     React.useEffect(() => {
         return () => {
-            clearInterval(resend_interval);
+            window.clearInterval(resend_interval);
```
Comment (niloofar-deriv) [inline packages/components/src/components/send-email-template/send-email-template.tsx:42]: I'm not sure about these, why we changed this? I mean why we ae reading these methods from `window`?

File: packages/account/src/Components/sent-email-modal/sent-email-modal.jsx (hunk 1 above)
Comment (yauheni-deriv) [reply packages/account/src/Components/sent-email-modal/sent-email-modal.jsx:107]: thanks @iman-fs 
yeas, I didn’t find its usage

File: packages/components/src/components/send-email-template/send-email-template.tsx (hunk 2 above)
Comment (yauheni-deriv) [reply packages/components/src/components/send-email-template/send-email-template.tsx:42]: anyway it is method of window object. typescript asked me to use this method in this way :)

File: packages/components/src/components/icon/icons.js [hunk 3]
```diff
@@ -181,6 +181,7 @@ import './common/ic-bank.svg';
 import './common/ic-bell.svg';
 import './common/ic-black-warning.svg';
 import './common/ic-block.svg';
+import './common/ic-blocked-advertisers-barred.svg';
```
Comment (matin-deriv) [inline packages/components/src/components/icon/icons.js]: what is this for? 😄

File: packages/components/src/components/send-email-template/send-email-template.tsx (hunk 2 above)
Comment (matin-deriv) [reply packages/components/src/components/send-email-template/send-email-template.tsx:42]: @yauheni-kryzhyk-deriv make sure to add QA checklist in the parent card for this to be tested.

**Review by matin-deriv — COMMENTED** [review]
CircleCI is failing, please find the root cause.

**Review by matin-deriv — APPROVED** [review]
LGTM

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**